Description: ignore userwarning in tests
 Required to run tests from source with the package already installed.
 Else one gets check_version_conflict warning from pkg_resources.
--- a/tornado/test/runtests.py
+++ b/tornado/test/runtests.py
@@ -72,6 +72,7 @@ if __name__ == '__main__':
     # setuptools sometimes gives ImportWarnings about things that are on
     # sys.path even if they're not being used.
     warnings.filterwarnings("ignore", category=ImportWarning)
+    warnings.filterwarnings("ignore", category=UserWarning)
     # Tornado generally shouldn't use anything deprecated, but some of
     # our dependencies do (last match wins).
     warnings.filterwarnings("ignore", category=DeprecationWarning)
